<ui version="4.0" >
<class>MainFrame</class>
<widget class="QMainWindow" name="MainFrame" >
<property name="geometry" >
<rect>
	<x>0</x>
	<y>0</y>
	<width>763</width>
	<height>663</height>
</rect>
</property>
<property name="sizePolicy" >
<sizepolicy>
	<hsizetype>7</hsizetype>
	<vsizetype>7</vsizetype>
	<horstretch>0</horstretch>
	<verstretch>0</verstretch>
</sizepolicy>
</property>
<property name="windowTitle" >
<string>EventSummary</string>
</property>
<widget class="QWidget" name="centralwidget" >
<layout class="QVBoxLayout" >
	<property name="margin" >
	<number>9</number>
	</property>
	<property name="spacing" >
	<number>6</number>
	</property>
	<item>
	<widget class="QTabWidget" name="tabWidget" >
	<property name="enabled" >
	<bool>true</bool>
	</property>
	<property name="minimumSize" >
	<size>
		<width>0</width>
		<height>0</height>
	</size>
	</property>
	<property name="sizeIncrement" >
	<size>
		<width>0</width>
		<height>0</height>
	</size>
	</property>
	<property name="focusPolicy" >
	<enum>Qt::TabFocus</enum>
	</property>
	<property name="contextMenuPolicy" >
	<enum>Qt::DefaultContextMenu</enum>
	</property>
	<property name="layoutDirection" >
	<enum>Qt::LeftToRight</enum>
	</property>
	<property name="currentIndex" >
	<number>1</number>
	</property>
	
	<widget class="QWidget" name="tabQcReport" >
	<attribute name="title" >
		<string>QcReport</string>
	</attribute>
	</widget>
	
	<widget class="QWidget" name="tabQcAlert" >
	<attribute name="title" >
		<string>QcAlert</string>
	</attribute>
	</widget>
	
	<widget class="QWidget" name="tabQcOver" >
		<attribute name="title" >
			<string>QcOverview</string>
		</attribute>
	</widget>
	
</widget>
	</item>
</layout>
</widget>
<widget class="QStatusBar" name="statusbar" />
<widget class="QMenuBar" name="menuBar" >
<property name="geometry" >
	<rect>
	<x>0</x>
	<y>0</y>
	<width>763</width>
	<height>29</height>
	</rect>
</property>
<widget class="QMenu" name="menuView" >
	<property name="title" >
	<string>&amp;View</string>
	</property>
</widget>
<widget class="QMenu" name="menuOptions" >
	<property name="title" >
	<string>&amp;Options</string>
	</property>
	<addaction name="actionAutoSelect" />
</widget>
<addaction name="menuOptions" />
<addaction name="menuView" />
</widget>
<action name="actionAutoSelect" >
<property name="checkable" >
	<bool>true</bool>
</property>
<property name="text" >
	<string>Auto update</string>
</property>
<property name="iconText" >
	<string>Auto update</string>
</property>
<property name="toolTip" >
	<string>Update Qc Parameter automatically (F8)</string>
</property>
<property name="shortcut" >
	<string>F8</string>
</property>
</action>
</widget>
<resources/>
<connections/>
</ui>
